test/expected_dir/octave1.m

```diff
@@ -4,7 +4,7 @@
 octave	comment	## 3-clause BSD license appended to this file.
 octave	blank	
 octave	code	function varargout = toledolu(LU)
-octave	comment	  ## -*- texinfo -*-
+octave	comment	  ## (*- texinfo -*)
```

Comment:
I don't understand the reason for the change. According to https://www.gnu.org/software/octave/doc/interpreter/Single-Line-Comments.html#Single-Line-Comments, any line that starts with `#` or `%` is a single line comment. Preceding whitespace is ignored.

Multiline comments (block comments) are denoted by `#{ .... }#` and `%{ ... }%`